cmd/tailscale/cli/debug.go

```diff
@@ -41,6 +42,7 @@ import (
 	"tailscale.com/types/logger"
 	"tailscale.com/util/must"
 	"tailscale.com/wgengine/capture"
+	"tailscale.com/wgengine/monitor"
```

Comment:
Was going to say that perhaps the debugging should still live in `tailscaled` (and be accessed via a LocalAPI endpoint) to avoid adding extra dependencies to `tailscale`, but my #7177 already brought this package in 🙈

Reply:
Oh no! Let's keep the CLI small. It's been a constant battle. Hence depaware.

Reply:
I just need to use the type, I'll see about moving it behind an interface.

Reply:
Sent out #7438

Reply:
Okay, I'll have to rewrite a bunch of the portmap code in order to move this to LocalAPI since the various `Disable*` options and the `VerboseLogs` flag are unsynchronized global variables, so we can't modify them in `tailscaled` concurrently between the LocalAPI endpoint and the actual for-real portmapper 😬

Reply:
Is it required? Seems like you could have a mutex that you hold while you invoke the portmapper (which sets the globals), so that you don't have to worry about concurrent invocations.

Reply:
Okay, switched the debug preferences to an `atomic.Pointer` and then used that to override options while doing the debug run, and then moved the code to LocalAPI. PTAL?